src/Runner.ts

```diff
@@ -104,20 +110,16 @@ export class Runner {
 
   private configFile(configFile?: string, filePath?: string) {
     const configExplorer = cosmiconfig("unibeautify", {});
-    if (configFile) {
-      return configExplorer
-        .load(configFile)
-        .then(result => (result ? result.config : null))
-        .catch(error => {
-          Promise.reject(error);
-        });
-    }
-    return configExplorer
-      .search(filePath)
+    const promise = configFile
```

Comment:
Should probably rename `promise` to something more meaningful. Even `loadConfigPromise` or something.